checker/jtreg/nullness/issue824/Class1.java

```diff
@@ -0,0 +1,20 @@
+/*
+ * @test
+ * @summary Test case for Issue 824 https://github.com/typetools/checker-framework/issues/824
+ * @compile -XDrawDiagnostics -Xlint:unchecked Class1.java
+ * @compile -XDrawDiagnostics -Xlint:unchecked -processor org.checkerframework.checker.nullness.NullnessChecker -AprintErrorStack -Anomsgtext Class2.java -Astubs=Class1.astub
```

Comment:
It would be nice to ensure that the correct errors are produced when the stub file is not given. I.e. have a copy of line 5 without the stub option and expected errors.